WooCommerce/UITestsFoundation/Screens/MyStore/MyStoreScreen.swift

```diff
@@ -8,19 +8,13 @@ public final class MyStoreScreen: ScreenObject {
     // TODO: Remove force `try` once `ScreenObject` migration is completed
     public let periodStatsTable = try! PeriodStatsTable()
 
-    private let settingsButtonGetter: (XCUIApplication) -> XCUIElement = {
-        $0.buttons["dashboard-settings-button"]
-    }
-
-    private var settingsButton: XCUIElement { settingsButtonGetter(app) }
-
```

Comment:
In my case, the `My Store` section still contains the `Settings` button, so restoring this code allowed me to execute two (currently skipped) login tests. Is it not shown for you?

Reply:
Yeah this is a bit complicated, so in this particular branch, it's still showing because it doesn't have the latest changes on `trunk` (I failed to properly rebase and the changes have accumulated 😞) so what I did was run the changes on the branch on CI which is latest trunk. 

This is [the PR](https://github.com/woocommerce/woocommerce-ios/pull/5885) that removes the settings icon on My Store screen which has been merged.

Reply:
Got it, thanks!